docs/api/tests.routing.bgp.md

```diff
@@ -18,3 +18,4 @@ anta_title: ANTA catalog for BGP tests
       filters:
         - "!test"
         - "!render"
+        - "!_.*"
```

Comment:
Not sure if that's related but Inputs are showing empty:
![image](https://github.com/user-attachments/assets/704f0dc9-9116-4ce9-ad82-e6b31aff35ff)

Reply:
Works on my machine.
But this has no effect on the private members.

Reply:
Forget my last comment, I was using mike instead of mkdocs :)

Reply:
It is related, I fixed it